composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -17,7 +17,7 @@
 
 
 def set_batch_sequence_length(batch: Dict[str, Tensor], curr_seq_len: int, truncate: bool = True) -> Batch:
-    """Set the sequence length of the current batch.
+    """Set the sequence length of a batch.
 
     Changes the sequence length of all tensors in the provided dictionary
```

Comment:
Should also mention the CUDA OOM note below. Can avoid duplication by doing:
```
__doc__.= """
Your docstring here

{note}
}}}.format(note==cuda_oom_note)
```

in both places.

```

Reply:
I [attempted this](https://github.com/mosaicml/composer/pull/622/commits/1344c7e0c4e21bf032614d8b289d44f8a97a9092) and it caused really weird rendering problems. The documentation for `set_batch_sequence_length` didn't appear at all in the API reference, and links to it redirected to the method card, which only printed the method signature and not the actual docstring. It's possible I messed up the syntax though.

Reply:
ya, sadly you can't do the `__doc__ = "whatever".format()` pattern in functions. Only classes.